### ISSUE TITLE
feat(grid): support referenced ui-recipes

### ISSUE THREAD

**Issue by roar-larsen** [equinor/dm-core-packages#639]
The Grid plugin should support rendering UI-recipes that are referenced.

Example where the grid plugin is showing a list 👇

grid.entity.json:
```
"list": {
    "type": "CORE:Reference",
    "address": "dmss://DemoDataSource/$gridList",
    "referenceType": "link"
  },
```

list.entity.json:
```
{
  "_id": "gridList",
  "name": "List",
  "type": "./blueprints/List",
  "tyre": [
    {
      "name": "Foo",
      "type": "./blueprints/Task",
      "description": "a task on the list"
    }
  ]
}
```

**Comment by eoaksnes** [comment]
TODO: Add to view config, resolve reference? that controls if the reference should show or the actual content.

**Comment by soofstad** [comment]
Closed with #665
